### ISSUE TITLE
[GSL for Android] error: no include path in which to search for limits.h

### ISSUE THREAD

**Issue by will-misfit** [bytedeco/javacpp-presets#166]
I am building gsl for android-arm on Mac OSX but got en error. So I check the config.log and found this:
```
configure:4211: /Users/willhou/Desktop/android-ndk-r10e//toolchains/arm-linux-androideabi-4.9/prebuilt/darwin-x86_64/bin/arm-linux-androideabi-cpp  conftest.c
In file included from /Users/willhou/Desktop/android-ndk-r10e/toolchains/arm-linux-androideabi-4.9/prebuilt/darwin-x86_64/lib/gcc/arm-linux-androideabi/4.9/include-fixed/syslimits.h:7:0,
                 from /Users/willhou/Desktop/android-ndk-r10e/toolchains/arm-linux-androideabi-4.9/prebuilt/darwin-x86_64/lib/gcc/arm-linux-androideabi/4.9/include-fixed/limits.h:34,
                 from conftest.c:13:
/Users/willhou/Desktop/android-ndk-r10e/toolchains/arm-linux-androideabi-4.9/prebuilt/darwin-x86_64/lib/gcc/arm-linux-androideabi/4.9/include-fixed/limits.h:168:61: error: no include path in which to search for limits.h
```

**Comment by saudet** [comment]
Which version? There's been some fixes since version 1.1, so please make sure to try with the latest code from the master branch.

**Comment by will-misfit** [comment]
I am using the latest code from the master branch. And I also tried it on Ubuntu, got the same error.

**Comment by saudet** [comment]
Yes, it appears the build was broken. It should be fixed now. Thanks for reporting!

**Comment by saudet** [comment]
Fixed in version 1.2. Again, thanks for reporting!
